runtime/testy/studentCode.py

```diff
@@ -8,10 +8,35 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  print("setup")
+  devices = Robot._getValue("hibike", "devices")
```

Comment:
ah crap. fixing.

the "refactor entire codebase to be consistent" has been on the list of things to do for a while now... 
it's my fault, i started the whole thing in camelCase when it should have been underscores x_x 
(i like underscores more too, but i thought that python style was camelCase. Yes, i know i'm an idiot)